src/main/java/androidx/core/view/View.kt

```diff
@@ -196,7 +197,12 @@ fun View.toBitmap(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Bitmap {
     if (!ViewCompat.isLaidOut(this)) {
         throw IllegalStateException("View needs to be laid out before calling toBitmap()")
     }
-    return Bitmap.createBitmap(width, height, config).applyCanvas(::draw)
+    return Bitmap.createBitmap(width, height, config).applyCanvas {
+        withSave {
```

Comment:
Since the canvas isn't re-used we don't need to bother with save/restoring.

Reply:
Yes, it's true. But with save/restore, we can keep code consistent with normal view draws. ^_^

Reply:
That consistency provides no value. You're doing work for no gain. Please remove.